node/resolvers/index.ts

```diff
@@ -140,8 +140,8 @@ export const QUERIES = {
       permissions
     }
   }`,
-  listUsers: `query users($organizationId: ID, $costCenterId: ID, $roleId: ID) {
```

Comment:
I don't think we can remove the `roleId` variable, as it is used by `node/resolvers/message.ts` to get a list of users that have a certain role. Can you explain why this solves the problem? Can you think of an alternative?